### PR TITLE
Expand project type support to all types of Vala resources

### PR DESCRIPTION
**Pull request by colinkiama**
This pull request mainly adds apps to the list but it also adds a few tools too.

### CODE REVIEW

**Comment by colinkiama** [comment]
@desiderantes I'm requesting your review because I've made huge changes and the repository's description needs also needs to be updated to reflect the new changes too (Repository supports any project types, not just libraries).

File: README.md [hunk 1]
```diff
@@ -1,83 +1,195 @@
...
+
+### Data Structures & Data Types
+
+- [Graphene](https://github.com/ebassi/graphene) - A thin layer of types for graphic libraries. It provides common types needed to handle 3D transformations: points, triangles, rectangles, quads, quaternions, vectors, matrices, spheres, etc.
+- [Libgee](https://wiki.gnome.org/Projects/Libgee) - A utility library providing GObject-based interfaces and classes for commonly used data structures (lists, maps, queues, trees, etc.).
+- [Numeric-GLib](https://github.com/arteymix/numeric-glib) - A collection of numeric data types for GLib (and Vala) via GCC extensions. It includes 128 bit integers & floats, complex types, vectorized operations, and decimal types.
+- [United](https://github.com/lcallarec/united) - A library for unit manipulation (like kilograms, meters, etc).
+- [vul (Vala Utility Libraries)](https://gitlab.gnome.org/BZHDeveloper/vul) - Vala Utility Libraries (Streams, Text, JSON, Archives).
```
Comment (desiderantes) [inline README.md:144]: I think this description should be improved. Sometimes projects just refuse to explain what they do.

**Comment by desiderantes** [comment]
This looks great!

**Comment by colinkiama** [comment]
I'm going to merge this in despite the failed CI check. The atom.io packages website is currently going through some server issues right now however the link does point to the right project.

**Pull request merged**